iquip/apps/scheduler.py

```diff
@@ -59,6 +81,22 @@ def __init__(self, parent: Optional[QWidget] = None):
         layout.addWidget(self.queueView)
 
 
+class ExperimentListView(QListView):
+    """Customized QListView class to detect right-click input.
+    
+    Signals:
+        rightButtonPressed(QMouseEvent): The information of the click input is sent.
```

Comment:
In the docstring for signals, we use intuitive argument names instead of their types, since the types are already in the codes right below.
For example, `rightButtonPressed(mouseEvent)`.